src/clang-cpp-frontend/optimized_includes/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+
+mangle(mangled_cpp "${CMAKE_SOURCE_DIR}/src/cpp/library"
+    RECURSIVE 1
+    MACRO ESBMC_FLAIL
+    ACC_HEADERS_INTO ${CMAKE_CURRENT_BINARY_DIR}/cpp_includes.h
+    LIST_DIRECTORIES FALSE)
```

Comment:
This is generating C symbols. They are required to be prefixed in order to be compatible with libraries, the system, etc. Please include a `PREFIX esbmc_libcpp_header_` or similar, see e.g. https://github.com/esbmc/esbmc/blob/db4d28d4bfdbb8a80f118c15ba6fa4f5f70ad0ea/src/c2goto/headers/CMakeLists.txt#L19-L22

Reply:
Done